test/ui/testAppActions.js

```diff
@@ -548,6 +548,41 @@ describe('appActions', function() {
 
   });
 
+  describe('hideUnavailableDevices', function() {
+    beforeEach(function() {
+      var state = appState.getInitial();
+      app.setState(state);
+    });
+
+    describe('[windows]', function() {
+      beforeEach(function() {
+        app.setState({_os: 'win'});
+      });
+
+      it('excludes only Tandem', function() {
+        expect(app.state.uploads.length).to.equal(10);
```

Comment:
I'm not crazy about hard-coding this, as it means every new device added will break this test. How about just testing that uploads.length decreases by the expected number after the hide function is called?